bin/quads-cli

```diff
@@ -262,30 +269,20 @@ def main(_args):
                 current = result['current']
                 new = result['new']
                 # now we need to know if we should wipe
-                url = os.path.join(api_url, 'lswipe')
-                data = {'cloudonly': new}
-                _response = requests.post(url, data)
-                wipe_js = _response.json()
-                wipe_value = '1'
-                if 'wipe' in wipe_js:
-                    if len(wipe_js['wipe']) == 0:
-                        print('ERROR')
-                        exit(1)
-                    for wipe in wipe_js['wipe']:
-                        wipe_value = wipe[new]
-                print('Moving ' + host + ' from ' + current + ' to ' + new + ', wipe = ' + wipe_value)
+                cloud = quads.get_clouds(name=new)
+                wipe_value = 1
+                if cloud:
+                    wipe_value = int(cloud["wipe"])
+                print('Moving %s from %s to %s, wipe = %s' % (host, current, new, wipe_value))
                 if not _args.dryrun:
                     try:
-                        if wipe_value == '1':
+                        if wipe_value == 1:
                             subprocess.check_call([_args.movecommand, host, current, new])
                         else:
                             subprocess.check_call([_args.movecommand, host, current, new, 'nowipe'])
```

Comment:
This should work fine as *any* command passed as a STDIN argument as the 4th command will result in nowipe.